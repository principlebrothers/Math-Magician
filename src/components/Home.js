import React from 'react';
import './style/Home.css';

function Home() {
  return (
    <div className="homeContainer">
      <header>
        <h1>Welcome to Maths magicians</h1>
      </header>
      <p>
        Math teaches us multiple problem-solving abilities like equations,
        computations, or algorithms which help us
        {' '}
        <br />
        further in multiple aspects of life. The most famous saying in math must be,
        “Everything around you is numbers, everything around you is mathematics”.
        <br />
        With math magicians calculation is simple and fast
      </p>
    </div>
  );
}

export default Home;
