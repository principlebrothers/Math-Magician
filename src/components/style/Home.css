body {
  background-color: rgba(88, 88, 88, 0.3);
}

.homeContainer {
  text-align: center;
}

.homeContainer header h1 {
  margin: 5rem 0;
  border-bottom: 2px solid rgb(78, 78, 78);
}

.homeContainer p {
  font-size: 2rem;
  padding: 2rem;
  line-height: 28px;
}
