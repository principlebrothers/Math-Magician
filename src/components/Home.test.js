import React from 'react';
import renderer from 'react-test-renderer';
import Home from "./Home";

test('Correctly render the home componenet',() => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
})