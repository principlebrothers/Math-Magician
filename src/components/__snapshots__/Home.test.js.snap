// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly render the home componenet 1`] = `
<div
  className="homeContainer"
>
  <header>
    <h1>
      Welcome to Maths magicians
    </h1>
  </header>
  <p>
    Math teaches us multiple problem-solving abilities like equations, computations, or algorithms which help us
     
    <br />
    further in multiple aspects of life. The most famous saying in math must be, “Everything around you is numbers, everything around you is mathematics”.
    <br />
    With math magicians calculation is simple and fast
  </p>
</div>
`;
